### PR TITLE
check version

### PR DESCRIPTION
**Pull request by sansx**
if there is a new version, inform the user of update #25

### CODE REVIEW

File: create-rust-app_cli/src/content/project.rs [hunk 1]
```diff
@@ -379,3 +380,15 @@ pub fn create_resource(backend: BackendFramework, resource_name: &str) -> Result
...
+
+pub fn check_cli_version() -> Result<()>{
+  let name = env!("CARGO_PKG_NAME");
+  let version = env!("CARGO_PKG_VERSION");
+  let informer = update_informer::new(registry::Crates, name, version);
+  logger::message(&format!("Welcome to use {} v{}", name, version));
+  if let Some(version) = informer.check_version().ok().flatten()  {
+    logger::message(&format!("New version {} is available!", version));
+    logger::message(&format!("If you want to update, try: {}", style("cargo install --force create-rust-app_cli").yellow()));
+  }
```
Comment (Wulf) [inline create-rust-app_cli/src/content/project.rs:392]: Minor suggestion for you!

I tried running `create-react-app` and noticed this message:

image

What do you think about something like this?

```suggestion
  if let Some(new_version) = informer.check_version().ok().flatten()  {
    logger::message(&format!("You are running `{name}` {version}, which is behind the latest release ({new_version})."));
    logger::message(&format!("If you want to update, try: {}", style("cargo install --force create-rust-app_cli").yellow()));
  } else {
    logger::message(&format!("You are running the latest version of {name} ({version})!"));
  }
```

**Review by Wulf — APPROVED** [review]
🚀 🚀 🚀 🚀 🚀 🚀 🚀 🚀 🚀 🚀

File: create-rust-app_cli/src/content/project.rs (hunk 1 above)
Comment (sansx) [reply create-rust-app_cli/src/content/project.rs:392]: it looks great, I make a update soon 😆

**Comment by Wulf** [comment]
Just released this :)

`create-rust-app_cli@5.1.1`

**Pull request merged**